src/dune_rules/dune_file.ml

```diff
@@ -475,6 +478,33 @@ module Mode_conf = struct
   end
 end
 
+module Ctypes = struct
+  type t =
+    { lib_name : string
```

Comment:
`Lib_name.t` perhaps?

Reply:
well, `lib_name` here is the name of the external/system library we're binding

Reply:
`Lib_name.t` applies to external libraries as well. `Lib_name.Local.t` is for local libraries.

Reply:
By external/system library I meant it's the name of the Ubuntu/Debian package that provides `/usr/lib/x86_64-linux-gnu/libmpg123.so.0.44.8`

It's collected in the dune file as an argument to pkg_config.  Though remember this isn't the final form, just an example to get something to build.